input/OFCB/1/Batched/1.xml

```diff
@@ -20,7 +20,7 @@
             <Source>
                 <outcommod>UOX</outcommod>
                 <outrecipe>UOX</outrecipe>
-                <throughput>1000</throughput>
+                <throughput>1e100</throughput>
```

Comment:
I just wanted to set an unlimited throughput for the source to avoid limit the filling of the reactor...